### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rhagarty**
- add some screenshots
- rearranged some items in README
- add ACKNOWLEDEMENTS.md file

### CODE REVIEW

File: ACKNOWLEDGEMENTS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Acknowledgements
+
+* Credit goes to
```
Comment (markstur) [inline ACKNOWLEDGEMENTS.md:3]: I think if you aren't going to give credit to anyone then you don't want this file, but I suppose it is W-I-P.

**Review by markstur — APPROVED** [review]
question the incomplete Ack file, but otherwise lgtm

File: README.md [hunk 2]
```diff
@@ -59,6 +59,24 @@ TBD!
 
 # Steps
 
+Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locally.
+
+## Deploy to IBM Cloud
+
+[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/sms-analysis-with-wks)
+
+1. Press the abovve ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
```
Comment (stevemar) [inline README.md:68]: above*

**Review by stevemar — APPROVED** [review]
i'll fix the two issues i saw

**Pull request merged**